tools/state-viewer/src/commands.rs

```diff
@@ -308,23 +308,31 @@ pub(crate) fn print_chain(
                 let mut chunk_debug_str: Vec<String> = Vec::new();
 
                 for shard_id in 0..header.chunk_mask().len() {
+                    let chunk_producer = runtime
+                        .get_chunk_producer(&epoch_id, header.height(), shard_id as u64)
+                        .unwrap();
                     if header.chunk_mask()[shard_id] {
                         let chunk = chain_store
                             .get_chunk(&block.chunks()[shard_id as usize].chunk_hash())
                             .unwrap()
                             .clone();
                         chunk_debug_str.push(format!(
-                            "{}: {} {: >3} Tgas ",
+                            "{}: {} {: >3} Tgas {: >10}",
                             shard_id,
                             format_hash(chunk.chunk_hash().0, show_full_hashes),
-                            chunk.cloned_header().gas_used() / (1024 * 1024 * 1024 * 1024)
+                            chunk.cloned_header().gas_used() / (1024 * 1024 * 1024 * 1024),
```

Comment:
Unrelated, but why is 1000 Tgas = 1024 x 1024 x 1024 x 1024 ?

Reply:
uh.. you're right ;-)